### PR TITLE
Fixing the toy image generator

### PR DESCRIPTION
**Pull request by MaxNoe**
When implementing the concentrations features, I found that the toy image generator was severely broken.

* The given intensity is not reproduced, the showers are much, much too bright. 

* Width and Length are taken as variance, not standard deviation

* The cleaning for the test event is far from optimal.

Here is the image that is used for the hillas tests:

example_after

The maximum pixel value is over 9000!
Just kidding, it's over 2000.

### CODE REVIEW

**Comment by MaxNoe** [comment]
This is the test image after the fix:
example_after

**Comment by MaxNoe** [comment]
The hillas tests are now failing because some methods report a `psi` rotated by 180 degrees.

File: ctapipe/image/tests/test_geometry_converter.py [hunk 1]
```diff
@@ -12,19 +12,33 @@
 cam_ids = CameraGeometry.get_known_camera_names()
 
 
+def create_mock_image(geom):
```
Comment (dneise) [inline ctapipe/image/tests/test_geometry_converter.py]: maybe one could expose the "magic numbers" as parameters of this function, like:
```
def create_mock_image(
    geom,
    centroid=0.3,
    width=0.03,
    length=0.10,
    psi="25d",
    intensity=0.5,
    nsb_level_pe=3,
):
```

Comment (dneise) [reply ctapipe/image/tests/test_geometry_converter.py]: or maybe one could somehow comment on this particular choice for these 6 parameters?

Comment (MaxNoe) [reply ctapipe/image/tests/test_geometry_converter.py]: The function is defined in the test, so is not a user facing function. The whole does also not work with astropy units. So if we would like to generalize this function, more work is needed anyway.

Comment (dneise) [reply ctapipe/image/tests/test_geometry_converter.py]: ok

**Comment by MaxNoe** [comment]
The tests are passing again, because I coerced the psi angles in the test to the [-90, 90] interval. 

Maybe we should think about adding this in the hillas_functions themselves but we will also not keep 5 hillas_functions, right?

**Comment by MaxNoe** [comment]
Somehow the installation of pyqt fails on travis for python 3.6

**Pull request merged**